Adding back the translations

KirbyFan102 opened the pull request:
Jackathan's original reposity of these is gone. I believe my repository has the most up-to-date translations of the story, if you are interested in preserving them as well. There's also another repository by @nuclearPhoenixx that includes the Epistle 3 Afterthoughts as well as the 'corrected' version of the same. Please consider merging it as well, or at least adding the files to your own repository, so we can have the most complete version of Marc's post-HL2 contents!

Commit:
Update Epistle3_Corrected.md

Commit:
Update README.md

Commit:
Update Epistle3_Corrected.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Created proper Spanish translation

This translation has been made by me and CrixixHL.

Commit:
Update and rename Epistle3_Corrected.md to Epistle3_Corrected_DE.md

p0sixkillah committed:
v.1.0. — final, relase

Final version, copied from the latest approved draft.

p0sixkillah committed:
v.1.0. — final, release

Final version, copied from the latest approved draft.

p0sixkillah committed:
v.1.0. — final, release, change #0001: "Added Wanda Bree's name to the list"

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

p0sixkillah committed:
v.1.0. — final, release

Fixed typo in the commit description.

Commit:
Update Epistle3_Corrected_DE.md

Commit:
Merge pull request #17 from Cyumus/master

Created proper Spanish translation

Commit:
Update Epistle3_Corrected_DE.md

Commit:
Update Epistle3_Corrected_DE.md

Commit:
Update Epistle3_Corrected_DE.md

Commit:
Update Epistle3_Corrected_DE.md

Commit:
Create Epistle3_Corrected.md

NuclearPhoenixx committed:
Merge pull request #1 from Jackathan/master

Proper Spanish translation

NuclearPhoenixx committed:
#2 by MiYanni

Merged Pull Request #2 by MiYanni!

NuclearPhoenixx committed:
#1 by tW4r

Merged Pull Request #1 by @tW4r

NuclearPhoenixx committed:
#6 by jmui

Merged Pull Request #6 by @jmui

NuclearPhoenixx committed:
#10 by frenchiveruti

Merged Pull Request #10 by @frenchiveruti

NuclearPhoenixx committed:
Update README.md

NuclearPhoenixx committed:
#13 by dmn001

Merged pull request #13 by @dmn001

NuclearPhoenixx committed:
#12 by CharlieC3 

Merged Pull Request #12 by @CharlieC3

NuclearPhoenixx committed:
new badges!

Commit:
update README.MD

I shoulda removed this line a LONG time ago

Commit:
Merge branch 'master' into master

Commit:
Merge pull request #20 from Phoenix1747/master

German translation seems solid, I'll merge it in

Commit:
Update Epistle3_Corrected.md

Commit:
Update README.md

Added some more archive links

Commit:
Update README.md

Commit:
Update README.md

lucasosse committed:
translation to Brazillian Portuguese has begun, 2 paragraphs done

lucasosse committed:
Update Epistle3_Corrected _PTBR.md

lucasosse committed:
Update Epistle3_Corrected _PTBR.md

lucasosse committed:
proofread

p0sixkillah committed:
Rename Epistle3_Corrected.md to Epistle3_Corrected_RU.md

p0sixkillah committed:
Rename Epistle3_Original.md to Epistle3_Original_RU.md

p0sixkillah committed:
Rename README.md to README_RU.md

p0sixkillah committed:
Add files via upload

Commit:
Added translation into Italian

zackexplosion committed:
Create Epistle3_Corrected_ZH_TW.md

zackexplosion committed:
Update Epistle3_Corrected_ZH_TW.md

Commit:
Merge pull request #26 from zackexplosion/patch-1

Create Epistle3_Corrected_ZH_TW.md

Commit:
Merge pull request #25 from st3no/master

Added translation into Italian

Commit:
Merge pull request #21 from lucasosse/ptbr

Translation to Brazilian Portuguese

Commit:
Rename Epistle3_Corrected _PTBR.md to Epistle3_Corrected_PTBR.md

Commit:
Merge pull request #24 from p0sixkillah/merge-friendly

Russian translation

janjilecek committed:
czech translation / cesky preklad (MarcLaidlaw-Epistle3)

Commit:
Merge pull request #27 from janjilecek/master

czech translation / cesky preklad (MarcLaidlaw-Epistle3)

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

NuclearPhoenixx commented:
If this repo is inactive, I can unarchive mine and you can create a PR there if you want. Just let me know.

https://github.com/NuclearPhoenixx/Epistle3

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 commented:
> If this repo is inactive, I can unarchive mine and you can create a PR there if you want. Just let me know.
> 
> https://github.com/NuclearPhoenixx/Epistle3

Okay, sure! Thank you for the kind offer 😀

KirbyFan102 commented:
@NuclearPhoenixx, please make your repository available for pull requests as soon as you can, ok? 🙏

NuclearPhoenixx commented:
Sure thing, un-archived it.

NuclearPhoenixx commented:
@KirbyFan102 Still interested? Just rediscovered this PR and I'd like to archive my repo again.

KirbyFan102 commented:
> @KirbyFan102 Still interested? Just rediscovered this PR and I'd like to archive my repo again.

I'm really sorry, man, I meant to get back to you, but life has been really awful lately. I meant to add two more translations to the repo, I'll ping you when I'm done. I promise you'll be able to archive it again by the end of the month, okay? Again, genuinely sorry for the lack of communication

KirbyFan102 committed:
Create x

KirbyFan102 committed:
Create x

NuclearPhoenixx commented:
No worries, please take your time!

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Delete Epistolary Afterthoughts/x

KirbyFan102 committed:
Delete Translations/x

KirbyFan102 committed:
Rename README.md to Original_README.md

KirbyFan102 committed:
Rename README_RU.md to Original_README_RU.md

KirbyFan102 committed:
Rename Epistle3_Corrected.md to Translations/Epistle3_Corrected.md

KirbyFan102 committed:
Rename Epistle3_Corrected_CZ.md to Translations/Epistle3_Corrected_CZ.md

KirbyFan102 committed:
Rename Epistle3_Corrected_DE.md to Translations/Epistle3_Corrected_DE.md

KirbyFan102 committed:
Rename Epistle3_Corrected_ES.md to Translations/Epistle3_Corrected_ES.md

KirbyFan102 committed:
Rename Translations/README.md to README.md

KirbyFan102 committed:
Rename Epistle3_Corrected_IT.md to Translations/Epistle3_Corrected_IT.md

KirbyFan102 committed:
Rename Epistle3_Corrected_PTBR.md to Translations/Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Rename Translations/Epistle3_Corrected.md to Epistle3_Corrected.md

KirbyFan102 committed:
Rename Epistle3_Corrected_RU.md to Translations/Epistle3_Corrected_RU.md

KirbyFan102 committed:
Rename Epistle3_Corrected_ZH_TW.md to Translations/Epistle3_Corrected_ZH_TW.md

KirbyFan102 committed:
Rename Epistle3_Original_RU.md to Translations/Epistle3_Original_RU.md

KirbyFan102 committed:
Rename Original_README_RU.md to README_RU.md

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Corrected.md

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Corrected.md

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Add files via upload

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Original_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Corrected_PTBR.md

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Corrected_PTBR.md

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Original_PTBR.md

KirbyFan102 committed:
Update EpistolaryAfterthoughts_Original_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Update Epistle3_Corrected_PTBR.md

KirbyFan102 committed:
Add files via upload

KirbyFan102 commented:
@NuclearPhoenixx, well, my fork of this repository is as complete as I can make it. I've added several new translations and even made some myself. I will admit that when I tried to add things to your repository, I ran into a strange issue where it would simply create separate 'patches' - it didn't seem like the forking worked for some reason. Anyway, if you would like to add or merge [my repository](https://github.com/KirbyFan102/MarcLaidlaw-Epistle3) with yours, or suggest how I might circumvent the aforementioned issue, feel free. I should note that I did change some things...

For example, in the 'corrected' version of Epistolary Afterthoughts, Alyx's name was still spelled as Alex, so I edited it. And in my portuguese translation of the corrected Epistle, I changed the second mention of Lake Huron to Michigan, since it was referencing Aperture/Tocsin's location. I didn't change the English version, though, because I wasn't sure you'd be okay with it? But you can change it yourself if you so choose

Anyway, thank you so much for your patience while I was gone!

NuclearPhoenixx commented:
Nice, please open a PR in my repo and I'm going to look at the changes in detail.

https://github.com/NuclearPhoenixx/Epistle3
